docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -526,6 +545,7 @@ For `enable`, `disable`, `delete`, `edit`, and `duplicate` actions, a JSON objec
 - `attributes.results.updated`: Rule objects that were updated during the action's execution
 - `attributes.results.deleted`: Rule objects that were deleted during the action's execution
 - `attributes.results.skipped`: Rules that were skipped during the action's execution
+- `attributes.results.updated`: Returns rule objects that were schedule for manual rule runs during the `run` action execution
```

Comment:
We should incorporate this one into the `attributes.results.updated` at line 545. Probably we could mention that in case of `run` action we use this property to return rule objects that were schedule for manual rule runs.

Reply:
Resolved in [f031d1d](https://github.com/elastic/security-docs/pull/5589/commits/f031d1d96f090523d2c492fadc02157f65472327)